### ISSUE TITLE
Shape issue with data  in  tflearn.lstm

### ISSUE THREAD

**Issue by bandarikanth** [tflearn/tflearn#920]
my dataset in .Json

sample dataset:

{"intents": [
	{"tag": "greetings",
         "patterns": ["Hi","hai","Is anyone there?", "Hello", "Are you there","what is happening","how is it going","great to see you","Nice to see you"],
         "responses": ["hi tell me","Hello, thanks for visiting", "hi,Good to see you ", "Hi there, how can I help?","hi,welcome to retrieval chatbot"]
        },
        {"tag": "goodbye",
         "patterns": ["Bye","take it eassy","See you later","Goodbye","Catch you later","Farewell","Have a good day","alright then!","Im out!","Take care.","Talk to you later","Have a good one"],
         "responses": ["See you later, thanks for visiting", "bye,Have a nice day", "Bye! Come back again soon."]
        },
	{"tag": "goodnight",
         "patterns": ["good night"],
         "responses": ["goodnight, thanks for visiting", "bye goodnight", "Bye! good night.","good thnq"]

I build model like these:

net = tflearn.input_data(shape=[None, len(train_x[0])])
net = tflearn.embedding(net, input_dim=10000, output_dim=128)
net = tflearn.lstm(net, 128, dropout=0.8)
net = tflearn.fully_connected(net, 32)
net = tflearn.fully_connected(net, 32)
net = tflearn.fully_connected(net, len(train_y[0]), activation='softmax')
net = tflearn.regression(net, optimizer='adam', learning_rate=0.001,
                         loss='categorical_crossentropy')

# Define model and setup tensorboard
model = tflearn.DNN(net, tensorboard_dir='tflearn_logs')

![screenshot from 2017-10-03 12 39 14](https://user-images.githubusercontent.com/26737645/31113668-f2c207ac-a837-11e7-84ba-9e261c9a7a7f.png)

model is created but when testing for response it showing  Errors like input data must be 2 dimensional

help out
